pkg/contracts/src/CVStrategy.sol

```diff
@@ -963,4 +874,95 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
+            revert ProposalNotInList(proposalId);
+        }
+        if (proposal.proposalStatus != StrategyStruct.ProposalStatus.Active) {
+            revert ProposalNotActive(proposalId);
+        }
+        if (msg.value <= arbitrableConfig.collateralAmount) {
+            revert InsufficientCollateral(msg.value, arbitrableConfig.collateralAmount);
+        }
+
+        uint256 arbitrationFee = msg.value - arbitrableConfig.collateralAmount;
+
+        collateralVault.depositCollateral{value: arbitrableConfig.collateralAmount}(proposalId, msg.sender);
+
+        uint256 disputeId = arbitrableConfig.arbitrator.createDispute{value: arbitrationFee}(RULING_OPTIONS, _extraData);
+
+        proposal.proposalStatus = StrategyStruct.ProposalStatus.Disputed;
+        proposal.disputeId = disputeId;
+        proposal.disputeTimestamp = block.timestamp;
+        proposal.challenger = msg.sender;
+        disputeIdToProposalId[disputeId] = proposalId;
+        emit ProposalDisputed(proposalId, disputeId, msg.sender, arbitrationFee, context);
+    }
+
+    function rule(uint256 _disputeID, uint256 _ruling) external override {
+        uint256 proposalId = disputeIdToProposalId[_disputeID];
+        StrategyStruct.Proposal storage proposal = proposals[proposalId];
+
+        if (proposalId == 0) {
+            revert ProposalNotInList(proposalId);
+        }
+        if (proposal.proposalStatus != StrategyStruct.ProposalStatus.Disputed) {
+            revert ProposalNotDisputed(proposalId);
+        }
+
+        bool isTimeOut = block.timestamp > proposal.disputeTimestamp + arbitrableConfig.defaultRulingTimeout;
+
+        if (!isTimeOut && msg.sender != address(arbitrableConfig.arbitrator)) {
+            revert OnlyArbitrator();
+        }
+
+        if (isTimeOut || _ruling == 0) {
+            if (arbitrableConfig.defaultRuling == 1) {
+                proposal.proposalStatus = StrategyStruct.ProposalStatus.Active;
+            }
+            if (arbitrableConfig.defaultRuling == 2) {
+                proposal.proposalStatus = StrategyStruct.ProposalStatus.Blocked;
+            }
+            collateralVault.withdrawCollateral(proposalId, proposal.challenger, arbitrableConfig.collateralAmount);
+            collateralVault.withdrawCollateral(proposalId, proposal.submitter, arbitrableConfig.collateralAmount);
+        } else if (_ruling == 1) {
+            proposal.proposalStatus = StrategyStruct.ProposalStatus.Active;
+            collateralVault.withdrawCollateral(proposalId, proposal.submitter, arbitrableConfig.collateralAmount);
+            collateralVault.withdrawCollateral(
+                proposalId, address(registryCommunity.councilSafe()), arbitrableConfig.collateralAmount
+            );
+        } else if (_ruling == 2) {
+            proposal.proposalStatus = StrategyStruct.ProposalStatus.Blocked;
+            collateralVault.withdrawCollateral(
+                proposalId, proposal.challenger, (arbitrableConfig.collateralAmount * 3) / 2
```

Comment:
this will be rejected by vault since in the vault space dedicated to the challenger he only has 1* collateral amount so you dont have choice duplicate the withdrawCollateral from proposer

Reply:
Puting it back to unresolved just to avoid being forgotten